threading/channels.nim

```diff
@@ -286,19 +288,23 @@ template trySend*[T](c: Chan[T], src: T): bool =
   trySend(c, isolate(src))
 
 proc tryRecv*[T](c: Chan[T], dst: var T): bool {.inline.} =
-  ## Tries to receive a message from the channel `c` and fill `dst` with its value.
-  ## This returns immediately even if no message is found. Doesn't block.
+  ## Tries to receive an item from the channel `c` and fill `dst` with its value.
   ## 
-  ## This can fail for all sort of reasons, including a lack of messages in the channel
-  ## to receive or contention.
+  ## Returns immediately once the lock on the Channel is acquired and an attempt
+  ## at fetching an item from it was made. 
+  ## The proc will not wait for items to appear.
+  ## 
+  ## This will fail if the channel is empty.
```

Comment:
Do we want to use plain "fail" for procs returning bool? Fail implies an error, but any result is technically *success of the attempt made*. Some wording of "returns `false` if no message received due to empty channel" is required, I think.

Reply:
Very fair point, I'll adjust the wording!